taxonomy-editor-frontend/src/pages/editentry/ListTranslations.jsx

```diff
@@ -140,7 +144,26 @@ const ListTranslations = ({ nodeObject, setNodeObject }) => {
         }
       }
     });
-    setShownLanguages(allLanguageCodes);
+    // get shown languages from local storage if it exists else use all languages
+    try {
+      const shownLanguages = JSON.parse(localStorage.getItem("shownLanguages"));
+      if (shownLanguages) {
+        setShownLanguages(shownLanguages);
+        shownLanguages.forEach((languageCode) => {
+          if (
+            !otherLangTags.some((item) => item.languageCode === languageCode)
+          ) {
+            handleAddTranslation(languageCode);
```

Comment:
as said in the previous PR, let's modify the `handleAddTranslation` so it only adds something when it doesn't exist , the current implementation is overriding

Reply:
Done